runtime/calamari/src/lib.rs

```diff
@@ -915,7 +915,7 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPallets,
+	AllPalletsWithSystem,
```

Comment:
This basically reverses the execution of the hooks in construct_runtime! hook. Are you sure this is fine ?

Reply:
I followed acala's and polkadot's change, both are the same. And I did runtime upgrade, so it looks fine.

Reply:
Mind explaining why you resolved this? 

According to https://github.com/paritytech/substrate/pull/10043
> Though it is recommended to be careful about the order of definition of the pallet in construct_runtime. and probably reorganize and use AllPalletsWithSystem, so the order is the expected normal order and not in reversed (as before this PR).

We would also need to reverse the order of the session pallets here to keep the hook order consistent
https://github.com/Manta-Network/Manta/blob/c4859d124453c52b27ef7144edb1b25eccc47059/runtime/calamari/src/lib.rs#L861

Reply:
@stechu @ghzlatarev What do you think about this change?
Originally, it was
`
Before this PR AllPallets represented all pallets without system in reversed order
`

I did runtime upgrade, it looks fine.

Reply:
As @Garandor said, we should use `AllPalletsWithoutSystemReversed` if we keep hooks order consistent.

Reply:
@Garandor 
`// Collator support. the order of these 5 are important and shall not change. `
Generally, the module index is fixed, we don't change it once it's set. Pallet order is different from hook order.

Reply:
I think it's safe to use `AllPalletsWithSystem` because we don't have any logic that relies on the order of the hooks staying the same as before this PR.
And using the same order as the pallets are defined is definitely saner option - https://github.com/paritytech/substrate/pull/10043

Reply:
If a pallet A has a function which is used in pallet B's  `on_initialize` hook, then pallet A's `on_initialize` hook needs to be executed first. According to a problem the Kilt guys ran into and presented at Sub0 - https://youtu.be/uKwEBE77JEI?t=1285

Reply:
This needs to be looked into some more.